waku/node/peer_manager/peer_manager.nim

```diff
@@ -914,6 +967,7 @@ proc relayConnectivityLoop*(pm: PeerManager) {.async.} =
       await pm.manageRelayPeers()
     else:
       await pm.connectToRelayPeers()
+    discard pm.updateTopicsHealth() # we don't want to await for the callbacks to finish
```

Comment:
discarding because the completion of the app callbacks shouldn't stop the peer manager from establishing new connections

Reply:
In fact, I think we can have a separate `async` loop but running within the `topic_healt.nim` module